test/DynamoCoreWpfTests/Utility/DispatcherUtil.cs

```diff
@@ -22,11 +24,39 @@ public static void DoEvents()
         }
 
         /// <summary>
-        ///     Helper method for DispatcherUtil
+        /// Force the Dispatcher to empty it's queue every 100 ms for a maximum 2 seconds or until
+        /// the check function returns true.
         /// </summary>
-        /// <param name="frame"></param>
-        /// <returns></returns>
-        private static object ExitFrame(object frame)
+        /// <param name="check">When check returns true, the even loop is stopped.</param>
+        [SecurityPermission(SecurityAction.Demand, Flags = SecurityPermissionFlag.UnmanagedCode)]
```

Comment:
??

Reply:
I do know that the Dispatcher does call into unmanaged code when it processes its message queue

Reply:
I feel you can probably get rid of it all, I think security permissions system is removed.

Reply:
![Screenshot 2023-12-05 at 10 32 07 AM](https://github.com/DynamoDS/Dynamo/assets/508936/9f46b177-568f-4743-8812-b45390a44bf7)